### ISSUE TITLE
Permitir buscar por bolsas suspensas

### ISSUE THREAD

**Issue by braganholo** [gems-uff/sapos#150]
Na tela de alocação de bolsas, permitir buscar por bolsas suspensas.

**Comment by danielpcampagna** [comment]
Uma bolsa suspensa seria equivalente a uma bolsa inativa?

**Comment by braganholo** [comment]
Não. Veja que na tela de alocação de bolsas, é possível cadastrar períodos de suspensão para uma determinada bolsa (por exemplo, o aluno saiu para doutorado sanduíche). Ali tem um checkbox "suspensão ativa". Essas são as bolsas suspensas. Seria bom também deixar pesquisar por período de suspensão (por exemplo, todas as bolsas que ficarão suspensas até Maio de 2015, ou que foram suspensas em Agosto de 2014).

**Comment by braganholo** [comment]
Eu confesso que não me lembro dos detalhes mas acho que tinha algo
relacionado com ajudar a secretaria a se lembrar de reativar a bolsa (na
época não tínhamos notificações). Uma alternativa seria remover esse campo.
Assim passaríamos a nos basear apenas na data e adicionaríamos uma
notificação de reativação da bolsa.

A questão é como controlar se a bolsa já foi ou não reativada. Como
garantir que houve a reativação? Enquanto tem o checkbox ele permanece
clicado enquanto a bolsa não é reativada. Não tem risco de alguém ver a
notificação e “esquecer” de agir. Acho que vale conversar com o Plastino e
com a secretaria sobre o trâmite de reativação da bolsa pra ver se dá
realmente para tirar esse campo de “suspensão ativa”.

**Comment by SAASVUUV** [comment]
Pelo o que foi falado na reunião e no grupo do Whatsapp, entendi que a suspensão permanece ativa até o retorno do aluno à bolsa, após o retorno desse estudante a mesma é desativada. Então, fiz da seguinte maneira:

![Image](https://github.com/user-attachments/assets/769514ec-9905-4518-aafe-73e0b4eefb9a)
As opções são: 
.Todas(irá buscar todas as suspensões independente se estão ativas ou não)
.Sim(irá buscar somente as que possuem suspensão ativa)
.Não(irá buscar somente as que não possuem suspensão ativa)


Desse modo, há a opção de serem selecionadas somente as suspensões ativas, as que não estão ativas, ou então as duas ao mesmo tempo. Podendo ou não serem aplicadas as datas de início e fim da suspensão. O que acham?

OBS: Somente fará essa busca caso o primeiro checkbox tenha sido clicado.

**Comment by leomurta** [comment]
@braganholo melhor deixar a checkbox mesmo, pois há ação externa para reativar.

@SAASVUUV eu acho que seria melhor não ter a checkbox na busca e a combo de "Suspensão ativa?" Ficasse inicialmente vazia, assim como as datas. Nesse estado, esse critério não é levado em conta. Se preencher alguma data, passa a levar a data em conta. Se selecionar algo na combo, passa a levar o estado em conta. Como opções da combo, poderíamos ter "alguma" e "nenhuma".

**Comment by leomurta** [comment]
Obrigado @SAASVUUV . Tudo certo. Já está em produção.
